Unsupported operand types for + ("Union[int, float]" and "int")

`mypy` is complaining about [`math.py`](https://github.com/wsanchez/sample-klein-app/blob/53bd173949db7aac9cf9d74f9544c928e8146586/src/sample_klein_app/application/math.py) thusly:

```
src/sample_klein_app/application/math.py: note: In member "add" of class "Application":
src/sample_klein_app/application/math.py:54: error: Unsupported operand types for + ("Union[int, float]" and "int")
src/sample_klein_app/application/math.py: note: In member "subtract" of class "Application":
src/sample_klein_app/application/math.py:70: error: Unsupported operand types for - ("Union[int, float]" and "int")
src/sample_klein_app/application/math.py: note: In member "multiply" of class "Application":
src/sample_klein_app/application/math.py:85: error: Unsupported operand types for * ("Union[int, float]" and "int")
src/sample_klein_app/application/math.py: note: In member "divide" of class "Application":
src/sample_klein_app/application/math.py:101: error: Unsupported operand types for / ("Union[int, float]" and "int")
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,8 +24,5 @@ matrix:
     - python: 3.5
       env: TOXENV=py35-trial-coverage,codecov_publish
 
-  allow_failures:
-    - env: TOXENV=lint-mypy
-
 script:
   - tox

--- a/src/sample_klein_app/application/math.py
+++ b/src/sample_klein_app/application/math.py
@@ -2,7 +2,7 @@
 Math application
 """
 
-from typing import Union
+from typing import Union, Iterable
 
 from twisted.web import http
 from twisted.web.iweb import IRequest
@@ -51,7 +51,8 @@ class Application(object):
 
         @param b: A number to add to C{a}.
         """
-        return "{}".format(self.numberify(a) + self.numberify(b))
+        x = self.numberify(a) + self.numberify(b)  # type: ignore #see #21
+        return "{}".format(x)
 
     @router.route("/subtract/<a>/<b>")
     def subtract(self, request: IRequest, a: str, b: str) -> KleinRenderable:
@@ -67,7 +68,8 @@ class Application(object):
 
         @param b: A number to subtract from C{a}.
         """
-        return "{}".format(self.numberify(a) - self.numberify(b))
+        x = self.numberify(a) - self.numberify(b)  # type: ignore #see #21
+        return "{}".format(x)
 
     @router.route("/multiply/<a>/<b>")
     def multiply(self, request: IRequest, a: str, b: str) -> KleinRenderable:
@@ -82,7 +84,8 @@ class Application(object):
 
         @param b: A number to multiply with C{a}.
         """
-        return "{}".format(self.numberify(a) * self.numberify(b))
+        x = self.numberify(a) * self.numberify(b)  # type: ignore #see #21
+        return "{}".format(x)
 
     @router.route("/divide/<a>/<b>")
     def divide(self, request: IRequest, a: str, b: str) -> KleinRenderable:
@@ -98,7 +101,8 @@ class Application(object):
 
         @param b: A number to divide C{a} by.
         """
-        return "{}".format(self.numberify(a) / self.numberify(b))
+        x = self.numberify(a) / self.numberify(b)  # type: ignore #see #21
+        return "{}".format(x)
 
     @router.handle_errors(ValueError)
     def valueError(self, request: IRequest, failure) -> KleinRenderable:

--- a/src/sample_klein_app/application/math.py
+++ b/src/sample_klein_app/application/math.py
@@ -2,7 +2,7 @@
 Math application
 """
 
-from typing import Union, Iterable
+from typing import Union
 
 from twisted.web import http
 from twisted.web.iweb import IRequest

